### PR TITLE
Features alignment

### PR DESCRIPTION
**Pull request by LucaMilanese90**
Fixes #

## Proposed Changes

  - https://github.com/oceanprotocol/market/pull/1958
  - https://github.com/oceanprotocol/market/pull/1921
  - align with `main` branch

### CODE REVIEW

File: src/@utils/docker.ts [hunk 1]
```diff
@@ -16,13 +16,10 @@ export async function getContainerChecksum(
...
   }
   try {
-    const response = await axios.post(
-      `https://dockerhub-proxy.oceanprotocol.com`,
-      {
-        image,
-        tag
-      }
-    )
+    const response = await axios.post(`https://dockerhub-proxy.delta-dao.com`, {
```
Comment (moritzkirstein) [inline src/@utils/docker.ts:19]: suggestion: we should put this into a new appConfig variable

File: src/components/Profile/History/index.tsx [hunk 2]
```diff
@@ -114,10 +115,12 @@ export default function HistoryPage({
     setRefetchJobs
   )
 
-  let defaultTabIndex = 0
-  defaultTab === 'ComputeJobs' ? (defaultTabIndex = 4) : (defaultTabIndex = 0)
```
Comment (moritzkirstein) [inline src/components/Profile/History/index.tsx:118]: suggestion: would be nice to retain this functionality (or re-enable it)
We may want to link to e.g., the compute history or the download tab and could achieve this by simply passing the tab index here for example

`.../profile?defaultTab=2` could link to compute history
```tsx
useEffect(() => {
  setTabIndex(defaultTab)
}, [defaultTab])
```
something like this

File: src/components/Profile/History/index.tsx [hunk 3]
```diff
@@ -66,6 +66,7 @@ export default function HistoryPage({
   const [refetchJobs, setRefetchJobs] = useState(false)
   const [isLoadingJobs, setIsLoadingJobs] = useState(false)
   const [jobs, setJobs] = useState<ComputeJobMetaData[]>([])
+  const [tabIndex, setTabIndex] = useState(0)
```
Comment (moritzkirstein) [inline src/components/Profile/History/index.tsx:69]: suggestion: see below, we may want to initialize based on query param

**Review by moritzkirstein — APPROVED** [review]

File: src/components/Profile/History/index.tsx (hunk 3 above)
Comment (LucaMilanese90) [reply src/components/Profile/History/index.tsx:69]: Implemented in https://github.com/deltaDAO/mvg-portal/pull/416

File: src/components/Profile/History/index.tsx (hunk 2 above)
Comment (LucaMilanese90) [reply src/components/Profile/History/index.tsx:118]: Implemented in https://github.com/deltaDAO/mvg-portal/pull/416

**Pull request merged**